llvm/docs/ReleaseNotes.rst

```diff
@@ -95,15 +95,45 @@ Changes to LLVM infrastructure
   `RFC  <https://discourse.llvm.org/t/rfc-ffat-lto-objects-support/63977>`_
   for more details.
 
+* ``ValueTypes.td`` is made richer to represent attributes for
+  ``MVT``.
+
+* ``Intrinsics.td`` is able to emit definitions of
+  ``TypeSig``. ``IntrinsicEmitter`` doesn't depend on
```

Comment:
Suggestion:

> ``IntrinsicEmitter`` no longer depends on ``MachineValueTypes.h``

The sentence you wrote is correct. But the suggestion is more idiomatic.

Reply:
Looking at this again, I think this should be split into 3 bullet points:

> * ``Intrinsics.td`` now emits ``TypeSig`` definitions
> * ``IntrinsicEmitter`` no longer depends on ``TypeSig``
> * ``IntrinsicEmitter`` no longer depends on  ``MachineValueTypes.h``